configuration/addons.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Installation of Add-ons
+
+Depending on the [package]({{base}}/configuration/packages.html) you have choosen during your first time setup, there are already some pre-installed add-ons.
+Additional add-ons can be installed in the different ways, described below.
+
+## Through Paper UI
+
+Navigate to the add-ons section.
+Search for the desired add-on in the categories and press install.
+
+![installing Add-ons through Paper UI](images/addons_index_installation_paperui.gif)
```

Comment:
Can you change this to `./images/addons_index_installation_paperui.gif`? (see my remark in #714)